docs/en/02_Developer_Guides/10_Email/index.md

```diff
@@ -136,9 +136,9 @@ Configuration of those properties looks like the following:
 use SilverStripe\Control\Director;
 use SilverStripe\Core\Config\Config;
 if(Director::isLive()) {
-    Config::inst()->update('Email', 'bcc_all_emails_to', "client@example.com");
+    Config::modify()->set('Email', 'bcc_all_emails_to', "client@example.com");
 } else {
-    Config::inst()->update('Email', 'send_all_emails_to', "developer@example.com");
+    Config::modify()->set('Email', 'send_all_emails_to', "developer@example.com");
```

Comment:
This will need to be `Email::class` too right?

Reply:
Gah